paper/joss_preview/paper.Rmd

```diff
@@ -0,0 +1,165 @@
+
+Each R package has a website with function documentation and a 'Get Started' guide.
+There are also several vignettes, including on `bboutools` [analytical methods](https://poissonconsulting.github.io/bboutools/articles/methods.html) and [prior selection](https://poissonconsulting.github.io/bboutools/articles/priors.html); `bbouretro` [analytical methods](https://poissonconsulting.github.io/bbouretro/articles/retro-methods.html); using [`bbousims` with `bboutools`](https://poissonconsulting.github.io/bbousims/articles/bboutools.html); and comparison of Bayesian and traditional method estimates with [empirical](https://poissonconsulting.github.io/bbousuite/articles/empirical-comparisons.html) and [simulated](https://poissonconsulting.github.io/bbousuite/articles/simulations.html) data. 
+
+![bboushiny GUI. \label{fig:bboushiny}](figures/bboushiny-survival.png)
+
+# Statement of need
+
+Boreal caribou are found in most provinces and territories in Canada and have been listed as threatened since 2003 [@ECCC_2023].
+Each jurisdiction has their own monitoring program responsible for boreal caribou.
+Numerous methods have been used to estimate population growth, which can make cross-jurisdictional conversations on the health of the species complex. 
+
+The suite of R packages in `bbousuite` aim to address this problem.
+`bboutools` provides a standardized methodology with simple, general models and sensible defaults for estimating survival, recruitment and population growth that can be compared across jurisdictions.
+In addition, `bboushiny` provides access to `bboutools` functionality in a GUI; `bbouretro` facilitates comparison with traditional methods; and `bbousims` facilitates comparison of various methods' ability to recover known parameter values from simulated data. 
+
+There is an existing web-based application for estimating boreal caribou population growth rate [@eacker_webbased_2019]. 
+However, users reported that it was difficult to use, especially if not familiar with Bayesian statistics, and had too many options to be considered a standarized method.  
+
+The `caribouMetrics` R package, and associated shiny app `BayesianCaribouDemographicProjection`, also have functionality to estimate boreal caribou demographic parameters.  
+However, the primary goal of these tools is to forecast boreal caribou population under different monitoring and disturbance scenarios [@dyson_2022].
+
+# Overview of methods
+
+Despite key methodological differences, which are explained in more detail below, `bboutools` and `bbouretro` use the same underlying equations to estimate survival, recruitment and population growth. 
+Survival is estimated from the monthly fate of collared adult females using the staggered entry Kaplan-Meier method [@pollock_survival_1989], with an option to include uncertain mortaliites in the total monthly mortalities summed prior to model fitting. 
```

Comment:
Does bboutools really use the Kaplan-Meier method?

Reply:
removed suggestion that the underlying methods are shared